README.md

```diff
@@ -23,9 +23,11 @@ The `.env` file is used by flask to set environment variables when running ` fla
 
 In order to run this application you will require the appropriate trello information in the `.env` file. They are as follows
```

Comment:
Minor, but Mongo rather than Trello now! It would be nice to also mention any Mongo setup required